### DISCUSSION TITLE
Next.js

### DISCUSSION THREAD

**Discussion by ayrtongomes** [system-ui/theme-ui#1152] [General] [answered]
Im starting with theme-ui and want to build a webapp using Next.js + theme-ui. 
Is theme-ui compatible with Next.js?
Do i need to be careful with something?

**Comment by lachlanjc** [comment, marked as answer]
Yep, it absolutely is compatible! Here’s the example we maintain for setting it up: https://github.com/system-ui/theme-ui/tree/master/examples/next

I also have my own starter using a custom theme for my org: https://github.com/hackclub/theme-starter

**Reply by ayrtongomes** [reply]
Awesome!

Thank u man!

**Reply by kivohin** [reply]
> Yep, it absolutely is compatible! Here’s the example we maintain for setting it up: https://github.com/system-ui/theme-ui/tree/master/examples/next
> 
> I also have my own starter using a custom theme for my org: https://github.com/hackclub/theme-starter

Thank you for this! 🙌🏼
